dali/operators/decoder/nvjpeg/nvjpeg_decoder_decoupled_api.h

```diff
@@ -618,7 +618,6 @@ class nvJPEGDecoder : public Operator<MixedBackend>, CachedDecoderImpl {
         [this, sample, &in, output_data, shape](int tid) {
           SampleWorker(sample->sample_idx, sample->file_name, in.size(), tid,
             in.data<uint8_t>(), output_data, streams_[tid]);
-          CacheStore(sample->file_name, output_data, shape, streams_[tid]);
```

Comment:
`shape` is unused now and Clang build fails due to unused lambda capture.

Reply:
done